function/go.sum

```diff
@@ -1,6 +1,5 @@
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
 cloud.google.com/go v0.34.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
-cloud.google.com/go v0.38.0 h1:ROfEUZz+Gh5pa62DJWXSaonyu3StP6EA6lPEXPI6mCo=
```

Comment:
Have you any idea why was file `go.sum` modify? I can't reproduce it locally

Reply:
I just ran `go mod tidy` and it made those changes.
`go.sum` is a generated file, so never edited manually.